flow/connectors/mysql/mysql.go

```diff
@@ -392,3 +392,30 @@ func (c *MySqlConnector) GetVersion(ctx context.Context) (string, error) {
 	}
 	return "", errors.New("failed to connect")
 }
+
+func (c *MySqlConnector) StatActivity(
+	ctx context.Context,
+	req *protos.PostgresPeerActivityInfoRequest,
+) (*protos.PeerStatResponse, error) {
+	rs, err := c.Execute(ctx, "select ID, COMMAND, STATE, TIME, INFO from performance_schema.processlist WHERE USER=?", c.config.User)
```

Comment:
You might expect this to have how long the connection has been open, but no, we only get how many seconds the connection has been in `STATE`

Reply:
looks like mariadb has [threads table](https://mariadb.com/docs/server/reference/sql-statements/administrative-sql-statements/system-tables/performance-schema/performance-schema-tables/performance-schema-threads-table)

Reply:
So does [MySQL](https://dev.mysql.com/doc/refman/8.4/en/performance-schema-threads-table.html), but with a different schema. Also seems less useful than processlist.